eloquent/ci-nightly-navigation2-prerequisites.yaml

```diff
@@ -0,0 +1,106 @@
+%YAML 1.1
+# ROS buildfarm ci-build file
+---
+build_environment_variables:
+  CMAKE_PREFIX_PATH: '/opt/ros/melodic:$CMAKE_PREFIX_PATH'
+  LD_LIBRARY_PATH: '/opt/ros/melodic/lib:$LD_LIBRARY_PATH'
+  NDDSHOME: '/opt/rti.com/rti_connext_dds-5.3.1'
+  PYTHONPATH: '/opt/ros/melodic/lib/python2.7/dist-packages:$PYTHONPATH'
+  ROS_PACKAGE_PATH: '/opt/ros/melodic/share'
+  ROS_PYTHON_VERSION: '3'
+  RTI_NC_LICENSE_ACCEPTED: 'yes'
+build_tool: colcon
+build_tool_args: '--cmake-args --no-warn-unused-cli'
+install_packages:
+- libasio-dev # for FastRTPS
+- libtinyxml2-dev # for FastRTPS
+- ros-melodic-common-msgs
+- ros-melodic-rosbash
+- ros-melodic-roscpp
+- ros-melodic-roscpp-tutorials
+- ros-melodic-roslaunch
+- ros-melodic-rosmsg
+- ros-melodic-rospy-tutorials
+- ros-melodic-tf2-msgs
+jenkins_job_label: ci-agent
+jenkins_job_priority: 50
+jenkins_job_timeout: 240
+jenkins_job_upstream_triggers:
+- nightly-fastrtps
+repos_files:
+- https://github.com/ros2/ros2/raw/navigation2/navigation2-prerequisites.repos
```

Comment:
Wouldn't it make more sense to use the latest 2.x tag which is 2.5.2? The `ros2` branch seems to be outdated...

Reply:
I just tested with 2.5.2 and it seems fine. Thanks for pointing out the problem. I'm making this change in the nav2 repo